specification/ai/Azure.AI.Client/main.tsp

```diff
@@ -39,7 +39,7 @@ namespace Azure.AI {
   title: "Azure AI",
 })
 @server(
-  "{endpoint}/{subscriptionId}/{resourceGroupName}/{projectName}",  // This URL is a placeholder only. It is not used by emitted SDKs. SDKs have custom code to contruct conneciton URLs for different operations.
+  "{endpoint}/agents/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{projectName}",  // This URL is for the Agents API.
```

Comment:
Would the V1.0 version change ?

Reply:
I don't think so. New service work is being done in a new Unified REST API. I think it's okay to hard-code it for now.